react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import Icon from '../../icons/Icon';
+
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText
+}) => (
+  <button className="ma__button-alert" onClick={onClick}>
+    <span className="ma__button-alert__show">{showText}</span>
+    <span className="ma__button-alert__hide">{hideText}</span>
+    <Icon />
+    {text}
+  </button>
+);
+
+ButtonAlert.propTypes = {
+  text: PropTypes.string,
+  class: PropTypes.string,
```

Comment:
This functionality is missing in the render of the component. you should append this prop to the default `ma__button-alert` class, if this prop has a value. `class` is usually ignored by react though, so we should probably rename it to something like...`classes`.
```suggestion
  classes: PropTypes.string,
```

Reply:
If this is accepted, classes could be added like:
```
const buttonClasses = 'ma__button-alert';
if (classes && classes.length > 0) {
  buttonClasses += ` ${classes}`;
}
```